APE_Black_Formatting.rst

```diff
@@ -0,0 +1,286 @@
+.. _Black: https://github.com/psf/black
+.. _Django DEP-0008: https://github.com/django/deps/blob/main/final/0008-black.rst
+.. _pandas pre-commit: https://pandas.pydata.org/docs/development/contributing_codebase.html#pre-commit
+.. _coding style: https://docs.astropy.org/en/latest/development/codeguide.html#coding-style-conventions
+
+Formatting Code with Black
+===========================
+
+authors: Tom Aldcroft, Erik Tollerud
+
+date-created: 2022 July 13
+
+date-last-revised: 2022 August 25
+
+date-accepted: <replace with accepted date>
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+This APE proposes that the `astropy core package
+<https://github.com/astropy/astropy>`_ adopt Black_ as the default code format.
```

Comment:
Is the word 'default' needed here, or what about just putting 'adopt Black for formatting code'?

Reply:
Will do.